src/redux/configureStore.js

```diff
@@ -0,0 +1,10 @@
+import { combineReducers, createStore } from 'redux';
+import statusHandler from './categories/categories';
+import bookHandler from './books/books';
+
+const rootReducer = combineReducers({
+  bookHandler,
+  statusHandler,
+});
+
+createStore(rootReducer);
```

Comment:
- [ ]  Please save the store in a variable and export it, So you can use the store in app components.
  - You can consider the below code.

``` ruby
const store = createStore(rootReducer);

export default store;